src/compiler/transformers/esnext.ts

```diff
@@ -101,11 +111,153 @@ namespace ts {
                     return visitParenthesizedExpression(node as ParenthesizedExpression, noDestructuringValue);
                 case SyntaxKind.CatchClause:
                     return visitCatchClause(node as CatchClause);
+                case SyntaxKind.PropertyDeclaration:

```

Comment:
why did we need to add these new cases? Weren't they visited before?

Reply:
The esnext transformer didn't need any special behavior with these SyntaxKinds until private names were introduced.